docs/3.0/field-options.md

```diff
@@ -327,3 +327,48 @@ Now, all fields will have the stacked layout throughout your app.
 :::option `use_resource`
 <!-- TODO: this -->
 :::
+
+:::option `components`
+The field's `components` option allows you to customize the view components used for rendering the field in all, `index`, `show` and `edit` views. This provides you with a high degree of flexibility.
+
+### Ejecting the field components
+To start customizing the field components, you can eject one or multiple field components using the `avo:eject` command. Ejecting a field component generates the necessary files for customization. Here's how you can use the `avo:eject` command:
+
+#### Ejecting All Components for a Field
+`$ rails g avo:eject --field-components <field_type>`
```

Comment:
Also, let's link to the eject command section for more details.

"For more advanced usage, see the `--field-components` page."